internal/genny/docker/templates/dot-dockerignore.tmpl

```diff
@@ -1,3 +1,4 @@
 node_modules/
+.pnp.*
```

Comment:
`.yarn` too maybe?

Reply:
Yeah I guess everything that is in the gitignore template should also be in dockerignore, per: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Reply:
I don't think it really matters. The [Dockerfile template](https://github.com/gobuffalo/cli/blob/v0.18.2/internal/genny/docker/templates/Dockerfile.tmpl) uses a multi-stage dockferfile where the assets and binary are built in a `builder` image and then copied over to the run image. The yarn files will only be present in the `builder` image once complete.

Reply:
Ah! right! I missed that. You are correct.